packages/did-ethr-resolver/src/utils/crypto.ts

```diff
@@ -52,9 +52,23 @@ export function hexify(value: string | object): string {
 * @private
 */
 export function addressOf(did: string): string {
-  const match = did.match(DIDPattern);
+  const match = did.match(didPattern);
   if (!match) {
     throw new Error('Invalid DID');
   }
   return match[1];
 }
+
+/**
+* Checks if did has a valid pattern, and returns the matched pattern array if it is
+*
+* @param did
+* @private
```

Comment:
Why this `@private` annotation?

Reply:
I looked at [crypto.ts](https://github.com/energywebfoundation/ew-did-registry/blob/eb2ebbc922a6aad792018a1db14c8230de190748/packages/did-ethr-resolver/src/utils/crypto.ts#L52) and it was used, so I thought to follow the same pattern. I guess, we can remove all `private` annotation.